### DISCUSSION TITLE
Google pay integration

### DISCUSSION THREAD

**Discussion by dalton5** [flutter-stripe/flutter_stripe#305] [Q&A]
Hi,

I use the payment sheet to propose the payment in my app. It works great.  

Now I would like to know how I can integrate Google pay. Are there big changes to do?

Is there a tutorial or code example?

Thanks

**Comment by DevFreeze** [comment]
Just add this package into your pubspec.yaml: https://pub.dev/packages/pay

Then add the google_pay_payment_profile.json file into the asset directory (there is an example in the example app). And set the variable from the PaymentSheet for `googlePay` to `true `

**Comment by dalton5** [comment]
I setup the json file with my merchent id and my stripe publishable key. I installed the pay package.

I added this to the androidManifest.xml

  <meta-data
    android:name="com.google.android.gms.wallet.api.enabled"
    android:value="true" />

Then I show the presentPaymentSheet with googlepay enabled but the button gpay on the top is not appearing.

What am I missing?

![image](https://user-images.githubusercontent.com/8339768/135757728-6fea85b7-4b2e-43aa-aea5-c4565e89e50d.png)

**Reply by anupin** [reply]
so i had the same issue and found out that you need to have google pay app on the phone with a payment method active in the phone, so you need a real card inserted in the google pay app this has to be done on a real phone
